test/csuite/timestamp_abort/main.c

```diff
@@ -97,10 +97,80 @@ usage(void)
 typedef struct {
 	WT_CONNECTION *conn;
 	uint64_t start;
-	uint32_t id;
+	uint32_t info;
 } WT_THREAD_DATA;
 
 /*
+ * thread_ts_run --
+ *	Runner function for a timestamp thread.
+ */
+static WT_THREAD_RET
+thread_ts_run(void *arg)
+{
+	WT_CURSOR *cur_stable;
+	WT_SESSION *session;
+	WT_THREAD_DATA *td;
+	uint64_t i, last_ts, oldest_ts;
+	int ret;
+	char tscfg[64];
+
+	td = (WT_THREAD_DATA *)arg;
+	last_ts = 0;
+
+	if ((ret = td->conn->open_session(td->conn, NULL, NULL, &session)) != 0)
+		testutil_die(ret, "WT_CONNECTION:open_session");
+	if ((ret = session->open_cursor(
+	    session, stable_store, NULL, NULL, &cur_stable)) != 0)
+		testutil_die(ret, "WT_SESSION.open_cursor: %s", stable_store);
```

Comment:
This can be more simply written as:
```
testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
testutil_check(session->open_cursor(session, stable_store, NULL, NULL, &cur_stable));
```